kong/conf_loader.lua

```diff
@@ -183,6 +183,18 @@ local function check_and_infer(conf)
                         "DCAwareRoundRobin policy is in use"
   end
 
+  for _, contact_point in ipairs(conf.cassandra_contact_points) do
+    local endpoint, err = utils.normalize_ip(contact_point)
+    if not endpoint then
+      errors[#errors+1] = "bad cassandra contact point '" .. contact_point ..
```

Comment:
The two error messages inconsistently capitalize "cassandra". Ideally, both should be lowercase.